SwiftLeeds/Views/About/ContributorsPage.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ContributorsPage.swift
```

Comment:
This should be ContributorsView rather than ContributorsPage to fit in with the other main view naming

Reply:
Well, Yeah for consistency. Done that.